spec/urlHash.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('URL manipulation methods', function() {
+
+  var UrlHash = require('../examples/lib/urlHash')
+  var urlHash;
+  var params = { who: 'me' }
+
+  beforeEach(()=>{
+    urlHash = UrlHash
+
+    spyOn(urlHash,'getUrlHashParameters')
+    spyOn(urlHash,'getUrlHashParameter')
+    spyOn(urlHash,'setUrlHashParameters')
+    spyOn(urlHash,'setUrlHashParameter')
+
+    urlHash.getUrlHashParameters()
+    urlHash.getUrlHashParameter('value')
+    urlHash.setUrlHashParameters(params)
+    urlHash.setUrlHashParameter('who','me')
```

Comment:
How about changing the who and me to something related like img and src so that the reader understands the use of the func?